login.css

```diff
@@ -0,0 +1,100 @@
+* {
+    padding: o;
+    box-sizing: border-box;
+    margin: 0;
+    color:  rgb(49, 49, 49);;
+}
+
+section {
+    margin: 3rem 1rem;
+    min-height: 94vh;
+    align-items: start;
+    width: 92vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+}
+.font {
+    font-size: 3rem;
+    font-weight: bold;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+
+#main-font {
+    font-size: 5.6rem;
+    font-weight: bolder;
+    color: #19A7CE;
+    white-space: nowrap;
+}
+
+@media only screen and (min-width: 768px) {

```

Comment:
Shouldn't it be 
     @media screen and (min-width: 768px)   ?

Reply:
Good point ☝️
Let me check on that